V1/DTOs/WalletTestResult/TestResultRequest.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using System;
 
-namespace CoviIDApiCore.V1.DTOs.TestResult
+namespace CoviIDApiCore.V1.DTOs.WalletTestResult
 {
     public class TestResultRequest
     {
         public string Key { get; set; }
-        public LaboratoryStatus LaboratoryStatus { get; set; }
+        public LaboratoryStatus LaboratoryStatus { get; set; } = LaboratoryStatus.Unsent;
         public ResultStatus ResultStatus { get; set; }
         public Laboratory Laboratory { get; set; }
         public string ReferenceNumber { get; set; }
         public DateTime TestedAt { get; set; }
+        public DateTime IssuedAt { get; set; }
```

Comment:
This is an internal field. It only gets added to the database when the test results have been verified with a lab